examples/with-sentry/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "with-sentry",
+  "version": "1.0.0",
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "next": "latest",
+    "raven-js": "^3.19.1",
```

Comment:
does this example work for SSR? I recall the node lib for sentry is "raven"

Reply:
It's only for client side. Please read comments above (in this PR).